tools/driver/p4c/p4c.site.cfg

```diff
@@ -0,0 +1,34 @@
+# -*- Python -*-
+
+import os
+
+def config_preprocessor():
```

Comment:
tofino should live in its backend and the install script should search the extensions folder for backend specific configurations and install them with the rest of the default configurations.